### ISSUE TITLE
[EXE] Weather Forecast Prediction Model

### ISSUE THREAD

**Issue by timothynn** [gimseng/99-ML-Learning-Projects#192]
#### Learning Goals
[Learning goals, bulleted/numbered list is preferred]
[e.g. learn the concept and the use of train/validation/test dataset using scikit-learn ]

### Exercise Statement
[Explain and describe what the exercise is]
[e.g. apply simple random-forest model to classify titanic survivability from titanic data ]

### Prerequisites 
[Prerequisites, in terms of concepts or other exercises in this repo]
[e.g. random-forest model, stochastic gradient descent, exercise #32]

### Data source/summary:
[Provide a succinct summary of what the data is and where it is from] 
[e.g. This involves covid19 fatality dataset from John Hopkin's website (links..) ]

### (Optional) Suggest/Propose Solutions
[e.g. I have the solution using PyTorch, will be happy to create pull request to include the exercise statement/solution]
[e.g. I think chapter 3 of A. Geron's textbook works out the solution for this exercise]
[e.g. fast.ai's chapter 5 has the perfect solution for this]

### (Optional) Further Links/Credits to Relevant Resources:
[e.g. This exercise and solution's proposal came from a lab session from DL2020]

**Comment by darpan-2001** [comment]
Hey @timothynn  can you give the source of dataset and the problem statement.
